src/code/circuit_python/motor.md

```diff
@@ -0,0 +1,232 @@
+and duty cycle.
+
+Frequency is the number of times each second that the signal changes from
+low to high and back to low. This is measured in Hertz (Hz).
+
+Duty Cycle is the fraction of the total pulse time where the signal is
+held high. 0% duty cycle is a signal that is always low. 100% duty cycle
+is a signal that is always high. 50% duty cyclce is a signal that spends
+an equal amount of time high and low.
+
+While PWM is an important concept to understand in robotics, we don't need
+to use it directly for controlling motors. We introduce this module here
+because it is used by the `adafruit_motor` module.
+
+API documentation for the `pwmio` module is available
+[here](https://docs.circuitpython.org/en/latest/shared-bindings/pwmio/index.html).
+
+## The adafruit_motor Module
+
+The `adafruit_motor` module provides tools for controlling a variety of
+types of motors. We will use the `adafruit_motor.servo` module specifically.
+
+Why would we use a "servo" module for controlling our motors? In our case,
+it's because they use the same control interface. The Vex MC29 makes our
+DC gear motors behave like "continuous rotation servos" by using PWM to
+control them. You've probably seen the common three-wire connector used by
+the Vex MC29 and servos in your kit. The red and black wires provide power
+to the servo/motor and the white wire carries a PWM signal to control the
+servo / motor.
+
+Servos control their position based on the signal they receive. We can
+control servos using the `Servo` class in `adafruit_motor.servo`. It has a
+property called `angle` that lets us set the position of the servo.
+Another property called `actuation_range` tells the `Servo` object how far
+the physical servo is able to move, so the code knows how to convert our
+angles into PWM signals.
+
+For controlling motors, we can use the `ContinousServo` class in
+`adafruit_motor.servo`. It has a property called `throttle` which controls
+the speed of the motor. `throttle` can be set to any value between -1.0
+and 1.0. Positive numbers will make the motor spin one direction and
+negative numbers will make the motor spin the other direction. Zero
+throttle will stop the motor.
+
+Both of these objects are created by giving them a `PWMOut` object
+connected to a pin. Optionally, you can also tell them the minimum and
+maximum pulse widths in milliseconds.
+
+API documentation for the `adafruit_motor` module is available
+[here](https://docs.circuitpython.org/projects/motor/en/latest/api.html).
+
+
+## Looping Over Numbers
+
+A common need in programs is looping over a range of numbers. This could
+be just because we need to repeat some action a certain number of times,
+or to loop through the elements in a list of items, for example. In
+Python, we can do this with a `for` loop and the `range` function.
+
+A `for` loop, in Python, runs the body of its loop once for each element a
+container. If you give it a list of numbers like `[1, 2, 3, 4]`, the loop
```

Comment:
no need for the quotative like here.  "If you give the list of numbers `[1, 2, 3, 4]` ..."